### ISSUE TITLE
web UI: add Play after current album

### ISSUE THREAD

**Issue by quite** [deluan/navidrome#627]
For the old-school album-listeners :+1:

**Comment by deluan** [comment]
You mean something like "Play Next", but instead of adding the selection as the next item to play, it adds the selection at the end of the current playing album?

**Comment by quite** [comment]
Yup. It would have to skip fast all the upcoming songs that are from the same album, and add selection after the last of those.

**Comment by quite** [comment]
Perhaps it adds clutter to the ui, how many ppl would use it...

**Comment by deluan** [comment]
Yeah, it may clutter the UI. I'll think about it. For now I'll keep this open, thanks for the suggestion.

**Comment by quite** [comment]
This is still relevant. As an album-listener who is currently listening to an album, I would like to queue up another album to listen to after the current one. 

You who liked this @zinggi and @crazy-ginger is your use-case something like this?

I have not used Navidrome for some time, and I don't know if there has been any changes to the UI so that this is feasible by now?

**Comment by Crazy-Ginger** [comment]
I'm still using Navidrom and would appreciate this feature. 

I think it would allow for users to create streams on the go without having to create playlists. Sometimes I listen to an initial album and then realise it ties nicely into another album or artist but don't want to create a temporary playlist that I'd end up deleting later

**Comment by quite** [comment]
@deluan do you really want people to spam issues just to keep them alive? Or alternatively not do that of course, and have them close -- or maybe even not filing issues at all! A bot marking them stale does not seem like good idea at this point.

**Comment by deluan** [comment]
The idea with the stale bot is to gauge the interest on old issues. Some were open a long time ago and the OPs are not using Navidrome anymore. 

Unfortunately this is the only way I can manage the issues at the moment. Working on Navidrome became a full time job I don't have time to dedicate to, so any way to reduce the workload, even if it at the expense of some user requests being missed, is welcome.

Regarding this issue, this is still something I want to have on Navidrome (and I'm marking it as "keep"), but will only be possible when I have time to work on the new player/UI, which it is progressing slower than I was hopping.

**Comment by Szeraax** [comment]
I'm going to take an opposite view: I think this would be a rarely used feature that clutters the UI and negatively impacts the UX (albeit, only to a minor degree). 

We already have play next (insert next in queue) and play later (insert at end of queue). Someone who wants to add a few albums should just use Play Later and call it good (similar mentality to making a playlist). Obviously, I won't complain if this does get added, but I doubt that I'll ever use it.
